app/src/main/res/layout/notification_dialog.xml

```diff
@@ -57,8 +57,8 @@
         android:id="@+id/btnconfirm"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="24dp"
-        android:layout_marginBottom="24dp"
+        android:layout_marginStart="@dimen/md"
+        android:layout_marginBottom="@dimen/md"
```

Comment:
Definitely not necessary. But in web dev, I've been learning to use a spacing utils, where we can have constants for space sizing. This may be useful to you to keep a solid flow when designing. Don't have to have 30dp, 32dp, 36dp, etc. throughout your app. You can create consistency through your design.

Reply:
Good idea.